xdn-vue-storefront-example/xdn/routes.js

```diff
@@ -0,0 +1,280 @@
+import { Router } from '@xdn/core/router'
+import { CACHE_ASSETS, CACHE_PAGES } from './cache'
+import { BACKENDS } from '@xdn/core/constants'
```

Comment:
@egormesyats is this public in the core API? This looks to be an internal constant of the framework and could be confusing to someone looking through the example.

Reply:
Yes. It's documented here: https://developer.moovweb.com/guides/react#section_configuring_the_xdn_router

Reply:
```suggestion
```